index.html

```diff
@@ -18,5 +18,157 @@
+              Maecenas pretium scelerisque tellus non vehicula.Sed et auctor libero.</p>
+            <p>Integer jgjhhh gravida sem et lobortis suscipit. Vivamus fringilla nisl vel neque rhoncus sodales. Suspendisse potenti. Vestibulum 
+              ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Cras ex nulla, condimentum at sagittis non, consectetur 
+              blandit ligula. Curabitur semper neque eget odio malesuada volutpat. Vivamus gravida imperdiet metus, tempor volutpat est venenatis nec. 
+              Curabitur egestas euismod fringilla. Nulla volutpat urna in odio finibus placerat. Aenean viverra ultrices bibendum. Duis non efficitur odio.</p>
+            <p>sit amet fermentum est iaculis sed. Vivamus dolor dolor, semper et mattis et, porttitor eu turpis. Quisque sed pharetra ligula. 
+              Vivamus elementum, nisi sit amet gravida pulvinar, orci libero dignissim eros, ac efficitur arcu dui id nibh. Cras sit amet vestibulum orci. 
+              Ut sed tempor arcu. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Nulla nec sapien justo. In ac 
+              vulputate nisi. Mauris bibendum ante nisl, ac sagittis metus lobortis id. 
+              <div class="read_more">
+              <a href="https://www.gardenersworld.com/offers/travel-and-events/bbc-gardeners-world-spring-fair/">
+              <button>Read More</button></a>
+            </div>
+          </article>
+
+        </section>
+        <!--article section ends here-->
+      </main>
+      <!--end of main part of the page ends here -->
+        
+      <footer>
+        <div id="about_us">
+            <h2 class="about_me">Who We Are?</h2>
+                <p class="information">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vehicula sem et dolor efficitur hendrerit. Duis rutrum justo vitae elit placerat varius. Nulla volutpat aliquet viverra. Vivamus semper facilisis viverra. Quisque aliquet varius dapibus. Fusce non tincidunt purus. Integer maximus magna leo, quis tincidunt mauris congue rutrum. Sed ultricies tempus nisl, non varius ipsum luctus id. Mauris leo ligula, semper ut aliquet vitae, porttitor ultrices diam. Donec vestibulum consectetur metus, eget pharetra risus.</p>
+                <p class="information">Praesent placerat vestibulum molestie. Sed euismod egestas mollis. Proin in eleifend lorem, vel tempor ipsum. Morbi volutpat enim ac feugiat eleifend. Nunc a dui egestas, tincidunt arcu ut, venenatis eros. Fusce fermentum risus quam, eget ultricies mi finibus quis. Maecenas sagittis erat in suscipit congue.</p>  
+          <div class="thank-you">
+            <p>Thank You for Visiting Our Blog.</p>
+          </div>          
+          <div class="trademark">
+            <small >&copy; Our Spring Garden-2021, by Konika</small>
+          </div>
+        </div>
+      </footer>
+  
+      
+    
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Take care with the formatting of your HTML - occasional blank lines can be useful to help make your code easier to read, but large blocks of blank space like this should be avoided

Reply:
Thank you for your comment. I rectified the issue as suggested . However every single review is showing outdated. I hope I am not very late to address the reviews. Kind regards.